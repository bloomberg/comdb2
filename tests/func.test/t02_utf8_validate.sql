SELECT utf8_valid('abc')
SELECT utf8_valid(x'616263FF')
