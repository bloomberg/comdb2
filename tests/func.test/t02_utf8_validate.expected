(utf8_valid('abc')=0)
(utf8_valid(x'616263FF')=-4)
